apps/judicial-system/web/src/components/AccordionItems/CaseFilesAccordionItem/CaseFilesAccordionItem.tsx

```diff
@@ -81,20 +81,24 @@ const CaseFilesAccordionItem: React.FC<Props> = (props) => {
           </Text>
           {canCaseFilesBeUploaded() && (
             <AnimatePresence>
-              {uploadState === UploadState.UPLOAD_ERROR && (
+              {(uploadState === UploadState.NONE_AVAILABLE ||
+                uploadState === UploadState.NONE_CAN_BE_UPLOADED ||
+                uploadState === UploadState.UPLOAD_ERROR) && (
```

Comment:
Show failure to upload message, if
- all remaining S3 links are broken before they are be uploaded to Auður (NONE_AVAILIBLE)
- a permanent upload error (case does not exist in Auður or unsupported file type) occurred during upload to Auður (NONE_CAN_BE_UPLOADED)
- a temporary error occurred during upload to Auður (UPLOAD_ERROR)